### PR TITLE
Disable ec2_key, ec2_snapshot and ec2_group tests

### PR DESCRIPTION
**Pull request by tremble**
##### SUMMARY

Work around #428 and #440 #441

##### ISSUE TYPE

- Feature Pull Request

##### COMPONENT NAME

ec2_key
ec2_snapshot
ec2_group

##### ADDITIONAL INFORMATION

### CODE REVIEW

**Review by jillr — APPROVED** [review]

**Comment by tremble** [comment]
recheck

**Comment by tremble** [comment]
recheck

**Comment by jillr** [comment]
recheck

**Comment by tremble** [comment]
recheck

**Comment by tremble** [comment]
recheck

File: tests/integration/targets/ec2_key/aliases [hunk 1]
```diff
@@ -1 +1,6 @@
+# reason: missing-dependency
+# We need either the openssl binary, pycrpto, or a compiler on the Py36 and Py38
+# Zuul nodes
```
Comment (pabelanger) [inline tests/integration/targets/ec2_key/aliases:3]: You can install this in CI by creating a top-level bindep.txt file with the following:

  openssl [test platform:rpm]
  
for example. https://docs.opendev.org/opendev/bindep/latest/ gives a few more examples.

Comment (tremble) [reply tests/integration/targets/ec2_key/aliases]: I've tried adding that file and it still failes:
https://dashboard.zuul.ansible.com/t/ansible/stream/08ad48a55d024770bc7e4766fb3caa82?logfile=console.log

**Comment by pabelanger** [comment]
I can see the issue, give me a little bit (today) to add depends-on here for jobs to fix.

**Comment by pabelanger** [comment]
recheck

**Comment by pabelanger** [comment]
recheck

**Comment by tremble** [comment]
recheck

**Comment by tremble** [comment]
recheck

**Comment by tremble** [comment]
recheck

**Comment by tremble** [comment]
Looks like with #427 (thanks @pabelanger) we're now able to install openssl, however something's still broken.

I'm going to disable the tests that are regularly failing (for now), so we can fix, retest, and re-enable them separately

**Pull request merged**